ts/output/chtml/fonts/tex.ts

```diff
@@ -60,6 +60,8 @@ import {delimiters} from '../../common/fonts/tex/delimiters.js';
 export class TeXFont extends
 CommonTeXFontMixin<CHTMLCharOptions, CHTMLVariantData, CHTMLDelimiterData, CHTMLFontDataClass>(CHTMLFontData) {
 
+    protected static defaultCssFamilyPrefix = 'MJXZERO, ';
```

Comment:
I'm not a big fan of putting the concatenation in the string. That presumes we always know that it will only ever be used with a some postfix.
Just in case the string is inadvertently used on its own somewhere, could that lead to a CSS error?

Reply:
I don't think so, but I've changed the usage as you requested.